includes/college-functions.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Responsible for college taxonomy related functionality
+ **/
+
+/**
+ * Displays a list of top degrees
+ * @author RJ Bruneel
+ * @since 3.0.0
+ * @param $term object | Object with top degrees
+ * @return string | Top Degrees List.
+ **/
+function display_top_degrees( $term ) {
+	$ret = "";
+	$top_degrees = get_field( 'top_degrees', 'colleges_' . $term->term_id );
+	if( $top_degrees ) :
+		foreach( $top_degrees as $top_degree ) :
+			$ret .= '<li><a href="' . $top_degree->post_name . '" class="text-inverse">' . $top_degree->post_title . '</a></li>';
+		endforeach;
+	endif;
+	return $ret;
+}
+
+/**
+ * Displays news title
+ * @author RJ Bruneel
+ * @since 3.0.0
+ * @param $term object | Object with title content
+ * @return string | News Title.
+ **/
+function display_news_title( $term ) {
+	if( $news_title = get_field( 'news_title', 'colleges_' . $term->term_id ) ) {
+		return $news_title . " News";
+	} else {
+		return str_replace( 'College of ', '', $term->name ) . " News";
+	}
+}
+
+/**
+ * Displays section content
+ * @author RJ Bruneel
+ * @since 3.0.0
+ * @param $section_id int | ID of the section
+ * @param $content string | Section content
+ * @return string | Section content surrounded by jumbotron if wrapper option checked.
+ **/
+function display_section( $section_id, $content ) {
```

Comment:
If you can use the section shortcode instead of calling this function, most of the work on background images and colors is already handled by the section display hooks.